content/hi/docs/setup/production-environment/_index.md

```diff
@@ -63,7 +63,7 @@ etcd को कॉन्फ़िगर करने और उपयोग क
 - *मल्टीपल कण्ट्रोल प्लेन सिस्टम बनाएं*: उच्च उपलब्धता के लिए, कण्ट्रोल प्लेन एक मशीन तक सीमित नहीं होना चाहिए। यदि कण्ट्रोल प्लेन सेवाएं एक init सेवा (जैसे systemd) द्वारा चलाई जाती हैं, तो प्रत्येक सेवा को कम से कम तीन मशीनों पर चलना चाहिए। हालाँकि, कुबेरनेट्स में पॉड्स के रूप में कण्ट्रोल प्लेन सेवाएं चलाना सुनिश्चित करता है कि आपके द्वारा अनुरोधित सेवाओं की प्रतिकृति संख्या हमेशा उपलब्ध रहेगी।
 अनुसूचक फॉल्ट सहने वाला होना चाहिए, लेकिन अत्यधिक उपलब्ध नहीं होना चाहिए। कुबेरनेट्स सेवाओं के नेता चुनाव करने के लिए कुछ डिप्लॉयमेंट उपकरण [राफ्ट](https://raft.github.io/) सर्वसम्मति एल्गोरिथ्म की स्थापना करते हैं। यदि प्राथमिक चला जाता है, तो दूसरी सेवा स्वयं को चुनती है और कार्यभार संभालती है।
 - *कई क्षेत्रों में विस्तार करना*: यदि अपने क्लस्टर को हर समय उपलब्ध रखना महत्वपूर्ण है, तो एक ऐसा क्लस्टर बनाने पर विचार करें, जो कई डेटा केंद्रों पर चलता हो, जिसे क्लाउड वातावरण में ज़ोन के रूप में संदर्भित किया जाता है। ज़ोन(zone) के समूहों को रीजन(region) कहा जाता है। एक ही क्षेत्र में कई क्षेत्रों में एक क्लस्टर फैलाकर, यह इस संभावना में सुधार कर सकता है कि एक क्षेत्र अनुपलब्ध होने पर भी आपका क्लस्टर कार्य करना जारी रखेगा।
-विवरण के लिए [Running in multiple zones](/docs/setup/best-practices/multiple-zones/) देखें।
+विवरण के लिए [एक से अधिक जोन मे चलाना ](/docs/setup/best-practices/multiple-zones/) देखें।
```

Comment:
```suggestion
विवरण के लिए [एक से अधिक जोन मे चलाना](/docs/setup/best-practices/multiple-zones/) देखें।
```

Reply:
IMO `ज़ोन` is more phonetically close to zone rather than `जोन`. Thoughts, WDYT @divya-mohan0209?

Reply:
Good catch, @Babapool . Please can you make the amendments @kundan2707 ?

Reply:
@divya-mohan0209  Yes i will make change